src/frontend/src/views/Upload/components/Samples/index.tsx

```diff
@@ -16,6 +16,7 @@ import {
 import { ParseErrors, Props } from "../common/types";
 import AlertTable from "./components/AlertTable";
 import Table from "./components/Table";
+import { strings } from "./strings";
```

Comment:
Similar to my other comment about the name `StringsType`: if our convention is to just call our per-pathogen set of strings the name `strings`, I think that is a bit vague. We might have other constant strings, possibly also being looked up on a different basis (eg, we look up different chunks of copywriting depending on if a user is a group admin or just a member). I think `strings.ts` is fine for the file: having a consistent bucket of strings adjacent to each component seems good. It's just the specific name of `strings` as the var we export for per-pathogen wording.

Reply:
Modified to name the variable `pathogenStrings`